.github/ISSUE_TEMPLATE/NOTABLE-CONTRIBUTION.md

```diff
@@ -0,0 +1,62 @@
+---
```

Comment:
I think we should use PRs only for notable contributions and discussions about their relevance.

A notable contribution should consist of a clear and concise commit title, ideally a single explicit line. Sometimes, it may need to include several links, but the main description should be in the actual contributions, such as the PR in the monorepo. A notable contribution entry here should not be the place where we spend too much time explaining why we believe it is significant.

If we want discussions, they can occur on the PR discussion page.